cardea-server/src/main/java/ca/on/oicr/gsi/cardea/server/service/CaseService.java

```diff
@@ -147,7 +147,12 @@ public TreeSet<ShesmuDetailedCase> getShesmuDetailedCases() {
 
   private Set<ShesmuSequencing> getSequencingForShesmuCase(Case kase) {
 
-    Set<ShesmuSequencing> sequencings = new HashSet<>();
+    TreeSet<ShesmuSequencing> sequencings =  new TreeSet<>(Comparator.comparing(ShesmuSequencing::getTest)
+        .thenComparing(shesmuSequencing -> String.join(",",
+            shesmuSequencing.getLimsIds().stream()
+                .map(ShesmuSample::getId)
+                .collect(Collectors.toList()))));
```

Comment:
Is the limsIds list guaranteed to come out in a defined order?

Reply:
Ah right - I see the value in having it as a TreeSet, but having it in a separate method from where the rest of the sorting happens makes things tricky. Maybe a comment here about how the limsIds are already sorted by ID, to help ourselves out later?